contributors/contributors/community.md

```diff
@@ -103,45 +86,45 @@ All contributors Level 2 and above are eligible for a [custom-printed Mattermost
+#### Documentation Contributors
 
 Documentation Contributors update, edit, and create new documentation, help text, training, and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/).
 
 Community Coordinator: Jason Blais and Justine Geffen
 
-#### [Quality Assurance Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Quality Assurance Contributors
 
 Quality Assurance Contributors help find, confirm and report product defects via our process at [https://www.mattermost.org/filing-issues/](https://www.mattermost.org/filing-issues/). Anyone who encounters a verifiable defect is welcome to file a bug on the system.
 
 Bonus: When QA Contributor finds and reports a bug on a release candidate build of Mattermost prior to the final release of the system, the can be awarded a [Mattermost Bug Hunter Award](https://forum.mattermost.org/t/mattermost-bug-hunter-awards/4979) for their contribution.
 
 Community Coordinator: TBD
 
-#### [Knowledge Base Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Knowledge Base Contributors
 
 Knowledge Base Contributors share feedback, questions, and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues, and the [Mattermost community server](https://community.mattermost.com/).
 
 Community Coordinator: Amy Blais
 
-#### [Deployment Solution Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### Deployment Solution Contributors
 
 Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.
 
 Community Coordinator: Jason Blais and Amy Blais
 
-#### [UX Design Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
+#### UX Design Contributors
 
 UX Design Contributors influence the design of the Mattermost user experience through discussion in the `ux-design` channel on the Mattermost continous integration server at [https://community.mattermost.com/core/channels/ux-design](https://community.mattermost.com/core/channels/ux-design) and through contributions to the [Mattermost User Experience Guidelines](https://docs.mattermost.com/developer/fx-guidelines.html) on [GitHub](https://github.com/mattermost/docs/blob/master/source/developer/fx-guidelines.rst).
```

Comment:
@andrewbrown00, is this link still good to use or should we link to the new design website?